packages/react-components/src/organisms/__tests__/TeamCreateOutputForm.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TeamCreateOutputForm from '../TeamCreateOutputForm';
+
+describe('TeamCreateOutputForm', () => {
+  test('click on button calls callback method', () => {
+    const spyOnCreate = jest.fn();
```

Comment:
This is a mock not a spy

Reply:
We're using this as a spy not as a full mock.

Mock functions are also known as "spies", because they let you spy on the behavior of a function that is called indirectly by some other code, rather than only testing the output.

Reply:
but a spy is a wrapper that will just intercept inputs and ouputs of a real function. This is just a jest.fn with no logic inside 😅

Reply:
https://jestjs.io/docs/mock-function-api